packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -234,7 +235,8 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
         :
         <Callout
           isBeakVisible={ false }
-          className={ css('ms-Dropdown-callout', styles.callout) }
+          className={ css('ms-Dropdown-callout', styles.callout,
+            calloutProperties ? calloutProperties.className : undefined) }
```

Comment:
Thanks! See [line 190 of BaseButton.tsx](https://github.com/OfficeDev/office-ui-fabric-react/blob/4d2ef75edb41b60f6d88bbf901e229ce2b04b3cd/packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx#L190) for an example of how those props are mixed in to the component:

```
<Icon { ...iconProps } className={ css(`${classNames.base}-icon`, classNames.icon, iconProps.className) } />
```

This is the pattern we want to follow here. Rather than pull only `className` out of `calloutProperties`, let's include all of the props so that anything can be overridden.